lib/formatter/csv/annotation_for_csv.rb

```diff
@@ -145,8 +145,8 @@ def answer_labels
               error_class:   "Task export error",
               error_message: "The task cannot be exported",
               context: {
-                classification: @classification,
-                annotation: @annotation,
+                classification: @classification.inspect,
```

Comment:
good point, this was fixing some older logging that zach put in that dumped the object instance data. I may just remove this as i am pretty sure this will fix it.